### ISSUE TITLE
Arbiter communication is not working service: [ERROR] [macaroon] no secret from arbiter: Invalid API key

### ISSUE THREAD

**Issue by Toshbrown** [me-box/core-export-service#25]
Hi, 

The export service currently can not communicate with the arbiter. Reported here https://github.com/me-box/databox/issues/78#issuecomment-321474229 and confirmed on my dev machine. 

```
Toshs-MacBook$docker ps
CONTAINER ID        IMAGE                      COMMAND                  CREATED             STATUS              PORTS 
6429cf83a5d8        export-service:latest      "./service"              1 hours ago        Up 1 hours         8080/tcp                                                             
da1ccd67e12b        arbiter:latest             "npm start"              1 hours ago        Up 1 hours         8080/tcp   
```                                                          

```
Toshs-MacBook$ d exec -t 6429cf83a5d8 cat /run/secrets/DATABOX_EXPORT_SERVICE_KEY
p5BDO3XZ4YYHclfg8ZMDdU8sbc0Ulw/lAnkChhfCGLk=
Toshs-MacBook$ d exec -t da1ccd67e12b cat /run/secrets/DATABOX_EXPORT_SERVICE_KEY
p5BDO3XZ4YYHclfg8ZMDdU8sbc0Ulw/lAnkChhfCGLk=
```

```
Toshs-MacBook-$ docker service logs databox_export-service
databox_export-service.1.qsiq8cerhtye@moby    | service: [ERROR] [macaroon] no secret from arbiter: Invalid API key
```

@sevenEng @yousefamar can you take a look.

**Comment by yousefamar** [comment]
Hunch: API key should be gotten from filesystem rather than env var now? Does export service shim it? https://github.com/me-box/core-export-service/blob/master/src/macaroon.ml#L14

**Comment by yousefamar** [comment]
Ah, it does, nevermind. https://github.com/me-box/core-export-service/blob/master/src/export_env.ml#L25

**Comment by yousefamar** [comment]
It seems to me that [this line](https://github.com/me-box/core-export-service/blame/master/src/export_env.ml#L29) might be the culprit? Is it decoding the token from base64 and using the result in the headers? If so, then it shouldn't do that, since the base64 string itself is the token IIRC. That line is at least 3 months old though, so I'm not sure it's that, since we would have noticed earlier.

**Comment by sevenEng** [comment]
Got this guys. Let me look into this a bit. Thanks for the hints @yousefamar

**Comment by sevenEng** [comment]
closed via #26
